R/liger.R

```diff
@@ -1674,7 +1670,7 @@ SNF <- function(object, dims.use = 1:ncol(object@H[[1]]), dist.use = "CR", cente
 #'   for using fftRtsne -- only first time runTSNE is called) (default NULL).
 #' @param rand.seed Random seed for reproducibility (default 42).
 #' 
-#' @return \code{liger} object with tsne.coords slot set. 
+#' @return \code{liger} object with dr.coord["tsne"] slot set. 
```

Comment:
This should be `dr.coords` (happens in a couple other places too). While this is only documentation, it should be consistent with the actual slot name.